src/main/java/twilightforest/biomes/TFBiomeDecorator.java

```diff
@@ -158,13 +158,16 @@ protected void genDecorations(BiomeGenBase biome) {
         }
 
         // random features!
-        if (randomGenerator.nextInt(6) == 0) {
-            int rx = chunk_X + randomGenerator.nextInt(16) + 8;
-            int rz = chunk_Z + randomGenerator.nextInt(16) + 8;
-            int ry = currentWorld.getHeightValue(rx, rz);
-            if (ry < 75) {
-                TFGenerator rf = randomFeature(randomGenerator);
-                rf.generate(currentWorld, randomGenerator, rx, ry, rz);
+        // now with chance
+        if (TFFeature.getRandom(randomGenerator, TwilightForestMod.minorFeatureGenChance)) {
+            if (randomGenerator.nextInt(6) == 0) {
+                int rx = chunk_X + randomGenerator.nextInt(16) + 8;
+                int rz = chunk_Z + randomGenerator.nextInt(16) + 8;
+                int ry = currentWorld.getHeightValue(rx, rz);
+                if (ry < 75) {
+                    TFGenerator rf = randomFeature(randomGenerator);
+                    rf.generate(currentWorld, randomGenerator, rx, ry, rz);
+                }
```

Comment:
This new minorFeatureGenChance config option now compounds with an existing 1 in 7 chance of generating a minor feature. This makes the config option a bit misleading and limits our ability to do things such as making minor features more common.

Would it be possible to combine these random chances, such as by using 1/7 (~14.25%) as the default value of the config option?

Reply:
yes, I think it’s logical to combine and leave the default 14% in the config